pkg/modulewriter/modulewriter_test.go

```diff
@@ -695,15 +714,30 @@ func (s *MySuite) TestWriteDeploymentGroup_PackerWriter(c *C) {
 		Name:    "packerGroup",
 		Modules: []config.Module{testPackerModule},
 	}
-	testWriter.writeDeploymentGroup(testDeploymentGroup, testVars, deploymentDir)
+
+	testDC := config.DeploymentConfig{
+		Config: config.Blueprint{
+			BlueprintName:   "",
+			Validators:      nil,
+			ValidationLevel: 0,
+			Vars:            testVars,
+			DeploymentGroups: []config.DeploymentGroup{
+				testDeploymentGroup,
+			},
+			TerraformBackendDefaults: config.TerraformBackend{},
```

Comment:
nit. You can omit this declaration of `TerraformBackend` , it has no efffect, default value is the same.

Reply:
Done